grr/core/grr_response_core/lib/rdfvalues/osquery.py

```diff
@@ -10,13 +10,14 @@
 from typing import Text
 
 from grr_response_core.lib.rdfvalues import structs as rdf_structs
+from grr_response_core.lib.rdfvalues import client_fs as rdf_client_fs
 from grr_response_proto import osquery_pb2
 
 
-class OsqueryArgs(rdf_structs.RDFProtoStruct):
-  """An RDF wrapper class for the `OsqueryArgs` proto."""
+class OsqueryFlowArgs(rdf_structs.RDFProtoStruct):
+  """An RDF wrapper class for the `OsqueryFlowArgs` proto."""
 
-  protobuf = osquery_pb2.OsqueryArgs
+  protobuf = osquery_pb2.OsqueryFlowArgs
   rdf_deps = []
```

Comment:
This is a server-only RDF value, putting them under `grr/server/grr_response_server/rdfvalues` would be more appropriate.